charts/kyverno/templates/_helpers/_deployment.tpl

```diff
@@ -0,0 +1,8 @@
+{{/* vim: set filetype=mustache: */}}
+
+{{- define "replicaCountCheck" }}
+  {{- $value := . }}
+  {{- if eq $value 0 }}
+    {{ fail "Kyverno does not support running with 0 replicas. Please provide a non-zero value." }}
+  {{- end }}
+{{- end }}
```

Comment:
```suggestion
{{- define "kyverno.deployment.replicas" -}}
{{- if eq (default 1 .) 0 -}}
  {{- fail "Kyverno does not support running with 0 replicas. Please provide a non-zero value." -}}
{{- end -}}
{{- . -}}
{{- end -}}
```

Reply:
Done. 
I have not used `default` because when we pass 0 to the function, instead of using 0 as the value it uses 1.
But everything works fine, we are getting the desired functionality even when we don't set values such as replicaCount when we install the chart